src/components/Navrail/Navrail.tsx

```diff
@@ -105,27 +104,27 @@ export const Navrail: FC = () => {
   return (
     <div className="dark">
       <Sidebar collapsible="icon">
-        <SidebarHeader className={`${styles.Header}`}>
-          {state === 'expanded' && (
-            <SidebarMenuButton className={`${styles.Gem}`} asChild>
-              <div onClick={() => navigate(ROUTES.OVERVIEW)}>
-                <Gem />
-              </div>
-            </SidebarMenuButton>
-          )}
-
+        <SidebarHeader className="flex-row-reverse justify-between">
           <SidebarTrigger />
+          <Button
+            size="icon"
+            variant="ghost"
+            onClick={() => navigate(ROUTES.OVERVIEW)}
+            className={state === 'expanded' ? '' : 'hidden'}
```

Comment:
instead of conditionally rendering the button, this updates the css of the `Button` to hide if the sidebar is collapsed. Same outcome, just a little easier to read